README.md

```diff
@@ -164,6 +164,24 @@ Typically, linking your own VPC to the lambdas and not setting addition NAT gate
 `sam deploy --template-file deploy.yaml --stack-name <user-defined-stack-name> --region <aws region> --capabilities CAPABILITY_NAMED_IAM --parameter-overrides vpcId=<vpcID> Subnets=<Subnets> SenderEmail=<SenderEmail> RecipientEmail=<RecipientEmail> --SlackWebhookUrls=<comma-separated-slack-webhook-urls>`  
 More information about setting up Slack webhooks can be found [here](https://api.slack.com/messaging/webhooks)
 
+#### To set up the step function execution through a scheduled run using CloudWatch rule, follow the steps below:
+
+1. Go to DBRestoreStateMachineEventRule section in the template.yaml of the Trapheuse repo.
```

Comment:
is it possible to add an **optional** input so that the user can provide the cron rule instead of having to modify the template.yml?

Reply:
@namitad We can do that but the user has to modify other things in the template.yaml like the input param https://github.com/intuit/Trapheus/blob/23e86f095def8c78ad81c872bb9365a682ec5c07/template.yaml#L1154 and corresponding id https://github.com/intuit/Trapheus/blob/23e86f095def8c78ad81c872bb9365a682ec5c07/template.yaml#L1153 and add or remove targets depending on his instances. So I thought there is no use of just adding cron expression as input param. But let me know your thoughts on this

Reply:
@namitad what do you think ^^

Reply:
i think that seems about right. as long as we have this detailed in the readme 👍